src/Compilers/Core/Portable/PublicAPI.Shipped.txt

```diff
@@ -356,9 +357,11 @@ Microsoft.CodeAnalysis.CandidateReason.WrongArity = 5 -> Microsoft.CodeAnalysis.
 Microsoft.CodeAnalysis.CaseInsensitiveComparison
 Microsoft.CodeAnalysis.ChildSyntaxList
 Microsoft.CodeAnalysis.ChildSyntaxList.Any() -> bool
+Microsoft.CodeAnalysis.ChildSyntaxList.ChildSyntaxList() -> void
```

Comment:
I am not sure why this file is getting modified.

Reply:
The new version of the analyzers adds implicit struct constructors (that already existed) to the public API files.

Reply:
> ➡️ The new version of the analyzers adds implicit struct constructors (that already existed) to the public API files. The changes for the analyzer update are in one commit, and a second commit that updates the file to have the lines in the order the analyzer wants (for future maintenance).

I think changes like that should be moved into a separate PR

---
In reply to: [512858730](https://github.com/dotnet/roslyn/pull/48896#discussion_r512858730) [](ancestors = 512858730)

Reply:
I needed a new version of the analyzer to address a false positive in one of the earlier commits. Do you want me to split this pull request? I'm happy to exclude the last two commits with the understanding that a couple of warnings will be disabled in the change until a future PR updates the analyzers.

Reply:
I do not expect a PR that has a goal "Reduce allocation from data taken in local performance trace" to touch any PublicAPI.Shipped files.

---
In reply to: [512863263](https://github.com/dotnet/roslyn/pull/48896#discussion_r512863263) [](ancestors = 512863263)

Reply:
➡️ Will separate to a new PR.